core/state.go

```diff
@@ -13,78 +13,84 @@ import (
 
 // Operators
 
-// OperatorSocket is formatted as "host:dispersalPort;retrievalPort;v2DispersalPort"
-type OperatorSocket string
+// OperatorSocket is formatted as follows:
+// v1-only 	  "host:dispersalPort;retrievalPort"
+// v1-and-v2  "host:dispersalPort;retrievalPort;v2DispersalPort;v2RetrievalPort"
+// v2-only    "host:dispersalPort;retrievalPort;v2DispersalPort;v2RetrievalPort" (currently v2 only mandates running v1)
```

Comment:
This isn't a valid setup though, as v1 is required

Reply:
We need a v2-only flag to support isolation, i.e allowing validators to run v1 and v2 on separate instances.